client/my-sites/marketing/connections/service.jsx

```diff
@@ -162,8 +162,8 @@ export class SharingService extends Component {
 		if ( service ) {
 			if ( keyringConnectionId ) {
 				// Since we have a Keyring connection to work with, we can immediately
-				// create or update the connection
-				this.createOrUpdateConnection( keyringConnectionId, externalUserId );
```

Comment:
We should do this only if the feature flag is set, no?

Reply:
Yeah, I'll change this when the Feature flag is ready!